packages/@okta/vuepress-theme-prose/assets/css/components/_content.scss

```diff
@@ -57,29 +63,27 @@
   }
 
   .tree-nav {
-    width: 18%;
+    width: 22%;
+    overflow: scroll;
+    position: sticky;
+    top: 70px;
```

Comment:
content-nav might need to adjust its positioning too

Reply:
I've redone this part, please take a look.